connect_ext_ppr/service.py

```diff
@@ -85,18 +85,27 @@ def add_deployments(installation, listings, config, logger):
             logger.info(f"The following Deployments have been created: {dep_ids}.")
 
 
-def update_product(data, config, logger):
-    product_id = data['id']
+def process_ppr_from_product_update(data, config, context, client, logger):
     with get_db_ctx_manager(config) as db:
-        q = db.query(Product).filter_by(id=product_id)
+        q = db.query(Deployment).filter_by(product_id=data['id'])
```

Comment:
[COD-B] Didn't get that case here. In case when new product version is available we should regenerate the PPR for all the deployments of that product, not the only first.